packages/react/test/specs/components/TextArea/TextArea-test.tsx

```diff
@@ -0,0 +1,49 @@
+import * as React from 'react'
+import { mountWithProvider as mount } from 'test/utils'
+import TextArea from 'src/components/TextArea/TextArea'
+import { isConformant } from 'test/specs/commonTests'
+
+const testValue = 'test value'
+
+describe('TextArea', () => {
+  isConformant(TextArea)
+
+  describe('defaultValue', () => {
+    test('sets "defaultValue" as initial "value"', () => {
+      const wrapper = mount(<TextArea defaultValue={testValue} />)
```

Comment:
```suggestion
      const wrapper = mount(<TextArea defaultValue='foo' />)
```

Can we inline it? It's more obvious in more cases

Reply:
But it's also more prone to errors, if you have to specify the same text at several places, you can make a typo. In the worst case, a test would pass while component is broken.

Reply:
Ok, I replaced all the occurences of `testValue` by `foo`.

Reply:
At the same time it makes code less readable.

> But it's also more prone to errors, if you have to specify the same text at several places, you can make a typo.

It _should_ be the same only in single test, it's good to use [faker](https://www.npmjs.com/package/faker), actually.

Reply:
I am using `faker` now. Hopefully, the code is still readable 🤔